docs/release-notes/feature-checkpointGC.rst

```diff
@@ -0,0 +1,9 @@
+:orphan:
+
+**New Features**
+
+-  Kubernetes: Add a feature where Determined offers the users to provide custom Checkpoint GC pod spec.
+      This configuration is done using the ``task_container_defaults.checkpoint_gc_pod_spec`` field
+      within your ``value.yaml`` file. When a custom CheckpointGC pod spec is defined, it takes
+      precedence over the default CheckpointGC pod specifications. This provides flexibility to
```

Comment:
I would be extra clear about what "default CheckpointGC pod specfications" means

I would mention that previous it would use the experiments pod spec but it can be overriden to use a configurable one with this field